src/main/java/use_cases/DatabaseManager.java

```diff
@@ -346,6 +346,48 @@ private void insertPostDB(Post post) {
         }
     }
 
+    /**
+     * Deletes the user from the database if the user exists
+     * @param user
+     */
+    public void deleteUser(User user)
+    {
+        try
+        {
+            String query = "DELETE FROM `user_info` WHERE `username`=?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, user.getUsername());
+            preparedStatement.execute();
+
+        } catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Given an user objects changes all the attributes
+     * such that match the ones given in the parameter
+     * except the id
+     * @param user
+     */
+    public void editUser(User user){
+        try
+        {
+            String query = "UPDATE FROM `user_info` SET `username`= ?, `password`=?, `bio`=? " +
+                    "WHERE `username`=?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, user.getUsername());
+            preparedStatement.setString(2, user.getUsername());
```

Comment:
why are these all set to user.getUsername()? Do you think we could use user.getPassword(), user.getBio, etc?